crates/re_viewport/src/viewport.rs

```diff
@@ -429,9 +387,11 @@ impl<'a, 'b> egui_tiles::Behavior<SpaceViewId> for TabViewer<'a, 'b> {
         self.ctx.re_ui.design_tokens.tab_bar_color
     }
 
+    //TODO(ab): currently unused, use it when egui_tiles passes `tiles` so we can retrieve state
```

Comment:
but we do pass tiles by now..?

Reply:
Oh, forgot about those. I'm still not using them actually in instead factored that directly into `TabWidget`, because factoring it there would need replication of the very annoying `tile_id` -> `space_view` -> `is selected?` logic. I'll remove them altogether.